ui/objects/semantic-ui/js/objects/editform/components/MoreInfo.jsx

```diff
@@ -57,21 +57,6 @@ export const MoreInfo = ({ activeIndex, handleActive, category }) => {
             }
           />
         </Grid.Column>{" "}
-        <Grid.Column>
-          <VocabularyTreeSelectField
```

Comment:
Where did it go? Was it redundant with create form somehow?

Reply:
According to the review: "	„Další údaje“ – Přijde mi tam zbytečný „typ materiálu“. Ten hlavní vybírám při zakládání karty". Primary material type is selected in CreateForm(kov, sklo, keramika, textil). Secondary material types could be still selected for other materials for the rest of the parts of details of the object